evervault/http/requestintercept.py

```diff
@@ -72,6 +72,8 @@ def new_req_func(
             self,
             method,
             url,
+            ignore_if_exact=client_self.ignore_if_exact,
+            ignore_if_endswith=client_self.ignore_if_endswith,
```

Comment:
Is there going to be an issue here? This is changing the parameters of the function which is used to overwrite the requests library's request function

Reply:
Doesn't need to be passed so I'll remove it